assets/src/edit-story/components/panels/design/captions/captions.js

```diff
@@ -107,16 +132,15 @@ function CaptionsPanel({ selectedElements, pushUpdate }) {
     <SimplePanel
       css={highlight?.showEffect && styles.FLASH}
       name="caption"
-      title={__('Captions', 'web-stories')}
+      title={__('Caption and subtitles', 'web-stories')}
       isPersistable={!highlight}
     >
       {isMixedValue && (
         <Row>
-          <BoxedTextInput
+          <Input
             value={MULTIPLE_DISPLAY_VALUE}
             disabled
             aria-label={__('Filename', 'web-stories')}
-            clear
             onChange={() => handleRemoveTrack()}
```

Comment:
should users be able to upload one caption to multiple videos? The `upload` button does not display if multiple are selected.

Reply:
For now, just one is supported 👍